.github/workflows/consistency_checks.yml

```diff
@@ -0,0 +1,30 @@
+name: Consistency Checks
```

Comment:
Wonder if we should merge Check PR Labels into this, to reduce the number of workflows running. WDYT?

Reply:
Sure. I can do that tomorrow morning.

Reply:
Done.